exonum/src/sandbox/consensus.rs

```diff
@@ -203,7 +203,8 @@ fn test_disable_and_enable() {
 
     // A fail is expected here as the node is disabled.
     sandbox.assert_state(HEIGHT_TWO, ROUND_ONE);
-    let result = try_add_one_height(&sandbox, &sandbox_state);
+    // TODO: use try_add_one_height (ECR-1817)
```

Comment:
Looks like there's a bug in sandbox - it doesn't handle persistent mempool correctly. Created https://jira.bf.local/browse/ECR-1817 about that.